apps/prairielearn/assets/stylesheets/pageLayout.css

```diff
@@ -135,9 +133,10 @@ body:has(div.app-container) {
   text-transform: uppercase;
   font-weight: bold;
   border-bottom: 1px solid var(--bs-body-color);
-  padding-bottom: 0.5rem;
+  padding-bottom: 0.3rem;
   margin-bottom: 0.5rem;
   white-space: nowrap;
+  transition: 300ms;
```

Comment:
What does this transition do? Can you list the specific properties you want to transition?